doc/languages-frameworks/php.section.md

```diff
@@ -291,3 +291,10 @@ in {
   };
 })
 ```
+You can access the `php` and `vendor` attributes from a package constructed
```

Comment:
It is not clear what those attributes are supposed to be.

Reply:
I'll rephrase immediately.